test/packer_basic_example_test.go

```diff
@@ -51,6 +52,49 @@ func TestPackerBasicExample(t *testing.T) {
 	assert.True(t, amiIsPublic)
 }
 
+func TestPackerMultipleConcurrentAmis(t *testing.T) {
+	t.Parallel()
+	i := 0
+
+	// Build a map of 3 randomId <-> packer.Options, in 3 random AWS Regions
+	// then build all of these AMIs in parallel and make sure that there are
+	// no errors.
+	var identifierToOptions = map[string]*packer.Options{}
+	for i < 3 {
```

Comment:
Why not a normal `for i := 0; i < 3; i++`?

Reply:
I just saw that on the golang examples page - it seemed like fewer characters to type and was otherwise equivalent.

Reply:
But you set `i := 0` a couple lines above and do `i++` a few lines below... So # of characters is identical... but now spread over extra lines?

Reply:
changed to standard for loop. I guess I was really looking for syntax like: `for i := range(0,2) ...`